### ISSUE TITLE
Locale data has not been initialized, call initializeDateFormatting(<locale>).

### ISSUE THREAD

**Issue by Billaids** [ikicodedev/calendar_timeline#33]
Hi, since version 1.0.2-null-safety.0 I'm facing the following exception.

════════ Exception caught by widgets library ═══════════════════════════════════
The following LocaleDataException was thrown building:
Locale data has not been initialized, call initializeDateFormatting(<locale>).

```dart
import 'package:calendar_timeline/calendar_timeline.dart';
import 'package:flutter/material.dart';

class DatePick extends StatefulWidget {
  DatePick({Key? key, required this.date, required this.onChange})
      : super(key: key);
  final DateTime date;
  final Function(DateTime?) onChange;

  @override
  _DatePickState createState() => _DatePickState();
}

class _DatePickState extends State<DatePick> {
  @override
  Widget build(BuildContext context) {
    DateTime currentDate = DateTime.now();
    return CalendarTimeline(
      initialDate: widget.date,
      firstDate: (currentDate).subtract(Duration(days: 21)),
      lastDate: currentDate.add(Duration(days: 21)),
      onDateSelected: widget.onChange,
      leftMargin: 20,
      monthColor: Colors.blueGrey,
      dayColor: Colors.black87,
      activeDayColor: Colors.white,
      activeBackgroundDayColor: Colors.black87,
      dotsColor: Color(0xFF333A47),
      locale: 'de',
    );
  }
}
```

**Comment by ikicodedev** [comment]
What version did you use previously?

**Comment by Billaids** [comment]
> What version did you use previously?

I used 1.0.1-null-safety.0 before and upgraded this morning to 1.0.2-null-safety.0

**Comment by ikicodedev** [comment]
It should be resolved in version 1.0.3-null-safety.0
